README.md

```diff
@@ -1,2 +1,24 @@
 # hw01
 Repository for homework 1
+
+## Eigen Aoki
+
+Hello, my name is Eigen Aoki.
+
+![](http://www.taikolegacy.com/wp-content/uploads/2017/07/c_eigen_aoki-440x350.jpg)
```

Comment:
Nice! You can also add pics by saving a jpg in your repository, and calling it via its file path.